erp/jobs/import_centres_vaccination.py

```diff
@@ -0,0 +1,94 @@
+"Imports vaccination centers from open data: https://www.data.gouv.fr/fr/datasets/lieux-de-vaccination-contre-la-covid-19/"
+
+import requests
+
+from django.conf import settings
+
+from core import mailer
+from erp.mapper import RecordMapper
+from erp.models import Activite
+
+
+API_URL = "https://www.data.gouv.fr/api/1/datasets/lieux-de-vaccination-contre-la-covid-19/"
+
+def job(dataset_url='', verbose=False, report=False):
+    if not dataset_url:
+        dataset_url = _retrieve_latest_dataset_url()
+
+    json_data = _retrieve_json_data(dataset_url)
+
+    activite = Activite.objects.filter(slug="centre-de-vaccination").first()
+    if not activite:
+        raise RuntimeError("L'activité Centre de vaccination n'existe pas.")
+
+    errors = []
+    imported = 0
+    skipped = 0
+
+    if "features" not in json_data:
+        raise RuntimeError("Liste des centres manquante")
+
+    for record in json_data["features"]:
+        try:
+            erp = RecordMapper(record).process(activite)
+            if verbose:
+                print(f"- {erp.nom}\n  {erp.code_postal} {erp.commune_ext.nom}")
+            else:
+                print(".", end="", flush=True)
+            imported += 1
+        except RuntimeError as err:
+            if not verbose:
+                print("S", end="", flush=True)
+            errors.append(err)
+            skipped += 1
+
+    if verbose and len(errors) > 0:
+        print("Erreurs rencontrées :")
+        for error in errors:
+            print(f"- {error}")
+
+    print("Opération effectuée:")
+    print(f"- {imported} centres importés")
+    print(f"- {skipped} écartés")
```

Comment:
How about [logging](https://docs.djangoproject.com/en/3.1/topics/logging/) instead of printing when we're in a scheduler context? Ideally we would print out when in a cli context and logging when in a scheduler one.

We're already doing that for [other jobs](https://github.com/MTES-MCT/acceslibre/blob/master/erp/jobs/check_closed_erps.py#L15).

Btw a cool feature with logging at the `error` level is that this will post to Sentry (like uncaught exceptions).